#!/usr/bin/python3

def raise_exception_msg(message=""):
    """Funciton to raise exception with given message.
    """

    raise NameError(message)
